### PR TITLE
Localazy 1.0.5

### PR DESCRIPTION
**Pull request by vaclavhodek**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [*] __I will not move an already shared step version's tag to another commit__
- [*] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [*] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [*] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [*] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/localazy/1.0.5/step.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  opts:
+    description: |-
+      Groups allow for running only selected rules from the configuration file.
+
+      For more information, visit [Localazy CLI - Command-Line Options](https://localazy.com/docs/cli/command-line-options#groups).
+    is_required: false
+    summary: The list of download groups separated by space
+    title: Groups
+- opts:
+    description: 'You should invoke Localazy CLI in the directory containing the configuration
```
Comment (ofalvai) [inline steps/localazy/1.0.5/step.yml:75]: Is this intentionally broken into two lines? I think you can remove the apostrophe and the extra whitespace at the end of the sentence.

**Review by ofalvai — CHANGES_REQUESTED** [review]
Just a minor comment, otherwise looks good! 🚀

**Comment by ofalvai** [comment]
Hello @vaclavhodek, apologies for the long silence on our part. I reviewed the PR and have a minor change request, otherwise looks good. Can you please close #3189? I'd like to merge this PR if you agree.

File: steps/localazy/1.0.5/step.yml (hunk 1 above)
Comment (vaclavhodek) [reply steps/localazy/1.0.5/step.yml:75]: There are some extra whitespaces, but it's not broken on our side. Is it possible that your tool made some changes to formatting?

**Comment by vaclavhodek** [comment]
Sorry, @ofalvai, I was also a bit busy and forget to check the PR.

I closed #3189 and it would be perfect if you can merge this PR. 

As for the comments about extra spaces & formatting, I will fix in the next version.

**Review by ofalvai — APPROVED** [review]

**Pull request merged**